Final_Project/Main.cpp

```diff
@@ -266,4 +282,29 @@ void terminateProgram()
 	CloseHandle(hIdleTimeoutThrd);
 	PostQuitMessage(0);
 	exit(1);
-}
\ No newline at end of file
+}
+
+/*-------------------------------------------------------------------------------------
+--	FUNCTION:	sendCharacter
+--
+--	DATE:			November 23, 2018
+--
+--	REVISIONS:		November 23, 2018
+--
+--	DESIGNER:		Dasha Strigoun, Kieran Lee, Alexander Song, Jason Kim
+--
+--	PROGRAMMER:		Alexander Song
+--
+--	INTERFACE:		void sencCharacter(HWND hwnd)
```

Comment:
This is very small, but there's a typo in sendCharacter.

Reply:
thanks for catching